app/src/main/AndroidManifest.xml

```diff
@@ -11,16 +11,16 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".MainActivity">
+        <activity android:name=".view.PostsViewActivity">
```

Comment:
Do you handle screen orientation changes? If no, please, specify the `screenOrientation="portrait"`